src/Makefile

```diff
@@ -17,19 +17,27 @@ DIRS = analyses \
        json-symtab-language \
        langapi \
        linking \
+       memory-analyzer \
        pointer-analysis \
        solvers \
        util \
        xmllang \
        # Empty last line
 
-all: cbmc.dir \
-     goto-analyzer.dir \
-     goto-cc.dir \
-     goto-diff.dir \
-     goto-instrument.dir \
-     goto-harness.dir \
-     # Empty last line
+ALLDEPS = cbmc.dir \
+          goto-analyzer.dir \
+          goto-cc.dir \
+          goto-diff.dir \
+          goto-instrument.dir \
+          goto-harness.dir \
+          # Empty last line
+
+
+ifeq ($(WITH_MEMORY_ANALYZER),1)
+  ALLDEPS += memory-analyzer.dir
+endif
```

Comment:
This introduction of `ALLDEPS` actually isn't even necessary as you can safely do
```
ifeq ($(WITH_MEMORY_ANALYZER),1)
all: memory-analyzer.dir
endif
```

Reply:
Changed.